chain/events/filter/index.go

```diff
@@ -364,6 +370,149 @@ func (ei *EventIndex) migrateToVersion4(ctx context.Context) error {
 	ei.vacuumDBAndCheckpointWAL(ctx)
 
 	log.Infof("Successfully migrated events to version 3 in %s", time.Since(now))
+	log.Infof("successfully migrated event index from version 2 to version 3 in %s", time.Since(now))
```

Comment:
```suggestion
	log.Infof("successfully migrated event index from version 3 to version 4 in %s", time.Since(now))
```

bad rebase I think, and removing the vacuum from this migration since you're doing it again in 5

Reply:
Right thanks